src/app/project/board/person-template/person-template.component.ts

```diff
@@ -1,12 +1,35 @@
-import { Component, Input } from '@angular/core';
-import { IPerson } from '../dataset-metadata';
+import { Component, Input, OnInit } from '@angular/core';
+import { Organization, Person } from '@dasch-swiss/dsp-js';
+
 
 @Component({
     selector: 'app-person-template',
     templateUrl: './person-template.component.html',
     styleUrls: ['./person-template.component.scss']
 })
-export class PersonTemplateComponent {
+export class PersonTemplateComponent implements OnInit {
     // input parameter
-    @Input() person: IPerson;
+    @Input() person: Person;
+
+    @Input() subProperties: Object;
+
+    organizationList = [];
+
+    ngOnInit() {
+        // check if members list is the list of [Organization] or [Iid]
+        let isOrganizationType: boolean = false;
```

Comment:
If property is initiated with value, type annotation is redundant. It's not wrong, but not necessary **for primitive types**.